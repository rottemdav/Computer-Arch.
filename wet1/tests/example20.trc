2 1 18 1 local_history global_tables using_share_mid
0x097e9904 T 0xc872970
0x44430 N 0x250
0x2a3948 N 0x0d8
0x44430 T 0x250
0x2f8 T 0x3e21130
0x878 N 0xa1bc18
0x38c0 T 0xd45ec
0x2c521ac N 0x21e8
0x7abe83c T 0x6a20e4c
0xa47f125c T 0xd3a62b00
0x44430 N 0x250
0x8c1554 N 0xa2944
0xf0d8 T 0xfd28
0x878 T 0xa1bc18
0x8316aa78 N 0x4d0ce8
0x998b04ac N 0x9be8
0xf2dd34 T 0xafd8d284
0xa6d9d0 N 0x99c8a0
0x38c0 N 0xd45ec
0x75cc0 T 0x45c
0xa47f125c N 0xd3a62b00
0x38c0 N 0xd45ec
0x7abe83c N 0x6a20e4c
0x38c0 N 0xd45ec
0x7abe83c N 0x6a20e4c
0x48a8e8 T 0x59a44
0x25c91a6c T 0x5f45580
0x878 N 0xa1bc18
0x7abe83c T 0x6a20e4c
0x44430 T 0x250
0x097e9904 T 0xc872970
0xa6d9d0 T 0x99c8a0
0xf2dd34 T 0xafd8d284
0x8316aa78 T 0x4d0ce8
0x600 N 0xb16f8088
0x75cc0 N 0x45c
0x25c91a6c N 0x5f45580
0xd8270 N 0x8b9cf94
0x878 T 0xa1bc18
0x48a8e8 N 0x59a44
0x03d84 N 0x73d78cc
0x600 T 0xb16f8088
0x59f5b8 N 0x9451274
0x48a8e8 T 0x59a44
0xa6d9d0 T 0x99c8a0
0x03d84 T 0x73d78cc
0x878 N 0xa1bc18
0x998b04ac T 0x9be8