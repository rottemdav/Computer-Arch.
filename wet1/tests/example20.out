0x97e9904 N 0x97e9908
0x44430 N 0x44434
0x2a3948 N 0x2a394c
0x44430 N 0x44434
0x2f8 N 0x2fc
0x878 N 0x87c
0x38c0 N 0x38c4
0x2c521ac N 0x2c521b0
0x7abe83c N 0x7abe840
0xa47f125c N 0xa47f1260
0x44430 N 0x44434
0x8c1554 N 0x8c1558
0xf0d8 N 0xf0dc
0x878 N 0x87c
0x8316aa78 N 0x8316aa7c
0x998b04ac N 0x998b04b0
0xf2dd34 N 0xf2dd38
0xa6d9d0 N 0xa6d9d4
0x38c0 N 0x38c4
0x75cc0 N 0x75cc4
0xa47f125c N 0xa47f1260
0x38c0 N 0x38c4
0x7abe83c N 0x7abe840
0x38c0 N 0x38c4
0x7abe83c N 0x7abe840
0x48a8e8 N 0x48a8ec
0x25c91a6c N 0x25c91a70
0x878 N 0x87c
0x7abe83c N 0x7abe840
0x44430 N 0x44434
0x97e9904 N 0x97e9908
0xa6d9d0 N 0xa6d9d4
0xf2dd34 N 0xf2dd38
0x8316aa78 N 0x8316aa7c
0x600 N 0x604
0x75cc0 N 0x75cc4
0x25c91a6c N 0x25c91a70
0xd8270 N 0xd8274
0x878 N 0x87c
0x48a8e8 N 0x48a8ec
0x3d84 N 0x3d88
0x600 N 0x604
0x59f5b8 N 0x59f5bc
0x48a8e8 N 0x48a8ec
0xa6d9d0 N 0xa6d9d4
0x3d84 T 0x73d78cc
0x878 N 0x87c
0x998b04ac N 0x998b04b0
flush_num: 23, br_num: 48, size: 104b
